src/multi_vector_simulator/utils/constants.py

```diff
@@ -265,16 +265,16 @@
         WARNING_TEXT: "allows defining the renewable share of the DSO supply (Values: Float). ",
         REQUIRED_IN_CSV_ELEMENTS: [ENERGY_PROVIDERS],
     },
-    REL_LOSSES: {
+    THERM_LOSSES_REL: {
         DEFAULT_VALUE: 0,
         UNIT: TYPE_FLOAT,
-        WARNING_TEXT: "allows considering relative thermal energy losses (Values: Float)",
+        WARNING_TEXT: "allows considering relative thermal energy losses (Values: Float). This is an advanced setting that most users can ignore.",
         REQUIRED_IN_CSV_ELEMENTS: [ENERGY_STORAGE],
     },
-    ABS_LOSSES: {
+    THERM_LOSSES_ABS: {
         DEFAULT_VALUE: 0,
         UNIT: TYPE_FLOAT,
-        WARNING_TEXT: "allows considering relative thermal energy losses (Values: Float)",
+        WARNING_TEXT: "allows considering relative thermal energy losses (Values: Float). This is an advanced setting that most users can ignore.",
```

Comment:
Exactly, we realized that this does not work and is required somewhere else. Hope it is in #718 now ;)